File: Nowcoder Multi-University/7/D.cpp
/*
 * @Date         : 2020-08-01 12:12:11
 * @Author       : ssyze
 * @Description  :
 */
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;
ll t, n;
int main()
{
    scanf("%lld", &t);
    while (t--) {
        scanf("%lld", &n);
        if (n == 1 || n == 24)
            puts("Fake news!");
        else
            puts("Nobody knows it better than me!");
    }
}

File: Nowcoder Multi-University/7/D.py
'''
@Date         : 2020-08-01 12:07:11
@Author       : ssyze
@Description  : 
'''
from math import *
t = int(input())
for ti in range(t):
    for n in range(1, 1000000000):
        sum = n * (n + 1) * (2 * n + 1) / 6
        sq = int(sqrt(sum))
        if sq * sq == sum:
            print("Fake news!")
            print(n)
